src/HTSPDemuxer.cpp

```diff
@@ -226,6 +226,47 @@ PVR_ERROR CHTSPDemuxer::CurrentSignal ( PVR_SIGNAL_STATUS &sig )
   return PVR_ERROR_NO_ERROR;
 }
 
+int64_t CHTSPDemuxer::GetTimeshiftTime() const
```

Comment:
in vnsi I only lock the cache because I don't want the gui to influence demuxer in any way. you don't want the gui lock out the demuxer a couple of times each cycle.